CMakeLists.txt

```diff
@@ -109,6 +120,11 @@ set(LAMINARD_CORE_SOURCES
     index_html_size.h
 )
 
+if (${CMAKE_SYSTEM_NAME} STREQUAL "FreeBSD")
+    include_directories(/usr/local/include)
+    link_directories(/usr/local/lib)
+endif()
```

Comment:
Surely this belongs in the ports file or in FreeBSD's general cmake configuration?

Alternatively, all the libraries used in `target_link_libraries` should instead be found via `find_library`, and then added to laminar using `target_link_libraries` and `target_include_paths`

Reply:
done